src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Region.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "region")
+data class Region(
+  @Id val id: Char?,
```

Comment:
let's match the region ids to the ones in delius instead of using our own "A,B,C,D" ids

Reply:
I picked this up from the spreadsheet David provided - I assumed that's what it was using.  I am unsure how to get these values from delius.